package.json

```diff
@@ -84,6 +84,7 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
     "helmet": "^2.1.2",
+    "html-loader": "^0.4.4",
```

Comment:
I think it still makes sense to separate the `index.html` to it's own file. We can remove `html-loader` in a separate PR if needed (unless anyone objects before merge time)